app/views/trains/index.html.slim

```diff
@@ -4,19 +4,20 @@ h1 Trains Index
   .alert.alert-success
     = notice
 
-table class="table"
-  thead
-    tr
-      - @trains.first.keys.each do |attribute|
-        th scope="col" = attribute.humanize
-      - if current_admin.present?
-        th scope="col" Actions
-  tbody
-    - @trains.each do |train|
+- if @trains.count > 0
```

Comment:
did we do this with any of our other tables?
i kinda like being able to see the headers.